CHANGELOG.md

```diff
@@ -2,19 +2,35 @@
 
 ## Unreleased
 
+## 3.6.0 (Feature release)
+
 ### New features
 
-#### Add display override for hiding content when printing
+#### Use colours with the tag component
+
+You can now [use colour with tags](https://design-system.service.gov.uk//tag/#using-colour-with-tags) to help distinguish between different tags - or to help draw the user’s attention to a tag if it’s especially important.
+
+This also means you should replace the `.govuk-tag--inactive` class with the `.govuk-tag--grey` class. `.govuk-tag--inactive` is now deprecrated, and it will be removed in a future release.
 
-Introduce a CSS class that can be used to hide any given element from the print view: `govuk-!-display-none-print`
+[Pull request #1711: Additional Tag modifier classes for different colours](https://github.com/alphagov/govuk-frontend/pull/1711).
+
+#### Hide elements when users print a page
+
+You can now hide elements when users print a page, using the new `govuk-!-display-none-print` class.
 
 [Pull request #1723: Add display override for hiding content when printing](https://github.com/alphagov/govuk-frontend/pull/1723).
 
+#### The `iff` Sass function is now deprecated
+
+You should no longer use the [`iff` Sass function](http://govuk-frontend-review.herokuapp.com/docs/#tools-function-iff). The function is now deprecated, and it will be removed in a future release.
```

Comment:
Non-blocking but we might not need to link to the docs from here, we generally don't surface them a lot. We could link to the code itself instead? https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/tools/_iff.scss#L14

Reply:
Agree on not linking to the Sass docs. I'm not a huge fan of linking directly to code as it can require quite a bit of untangling on the user's part. I've removed the link for now - I think the advice should be clear enough on its own, for users who have been using `iff`? (And anyone else can safely ignore it.)